From b3559a35154632c868195da9034b85e28b824e02 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:52:03 -0400
Subject: [tools] perf: Comment typo fix

Message-id: <1408953210-26343-236-git-send-email-jolsa@redhat.com>
Patchwork-id: 88966
O-Subject: [PATCH RHEL7.1 BZ1133083 235/322] tools perf: Comment typo fix
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 88aca8d966a1349631a0946ed77c7ed360519ed4
Author: Cody P Schafer <cody@linux.vnet.ibm.com>
Date: Wed Jan 8 08:43:51 2014 -0800

description
===========
s/temr/term/
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/pmu.c b/tools/perf/util/pmu.c
index 0934d64..d9cab4d 100644
--- a/tools/perf/util/pmu.c
+++ b/tools/perf/util/pmu.c
@@ -505,7 +505,7 @@ static __u64 pmu_format_value(unsigned long *format, __u64 value)
 
 /*
  * Setup one of config[12] attr members based on the
- * user input data - temr parameter.
+ * user input data - term parameter.
  */
 static int pmu_config_term(struct list_head *formats,
       struct perf_event_attr *attr,
-- 
1.7.1